src/component/User/Profile.js

```diff
@@ -1,10 +1,16 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect, useReducer, useMemo } from 'react';
 import Loader from '../layout/Loader/Loader';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faHeart } from '@fortawesome/free-solid-svg-icons';
 import { faSignOut } from '@fortawesome/free-solid-svg-icons';
-
 import NavigateButton from '../layout/NavigateButton/NavigateButton';
+import { initialState, userReducer } from '../../reducers/userReducer';
+import { loadUser } from '../../actions/userAction';
```

Comment:
Since you created `loadUser` action in another branch it makes it not possible to really check if the code works. I would wait till the other 2 PRs are merged with dev to check it.